model_compression_toolkit/gptq/common/gptq_config.py

```diff
@@ -88,7 +89,8 @@ def __init__(self,
                  norm_weights: bool = False,
                  quantizer_config: GumbelConfig = GumbelConfig(),
                  optimizer_quantization_parameter: Any = None,
-                 optimizer_bias: Any = None):
+                 optimizer_bias: Any = None,
+                 gumbel_norm: float = 0.5):
```

Comment:
rename gumbel_norm to gumbel_varinace or gumbel_scale (https://en.wikipedia.org/wiki/Gumbel_distribution)